### ISSUE TITLE
Blueberry (and Raspberry): Change default yield and add to static data

### ISSUE THREAD

**Issue by LolandaE** [bcgov/agri-nmp#627]
As a blueberry grower, I want the calculator to default to a reasonable yield, so that I don't have to do research if I do not know my current yield

![Image](https://github.com/bcgov/agri-nmp/assets/103149911/393c730b-7253-4aa6-8c6e-aca8bf2e4ea4)

**Additional Context**
- Only change the default yield for the blueberry pathway, on the Crop Details screen
- Add default yield to static data for both blueberry and raspberry - there should already be static data with default yield in the database

**Acceptance Criteria**
- [ ] The yield for blueberries defaults to 5 tons/ac on the Crop Details screen, instead of the current default of 7 tons/acre
- [ ] The default yield is added to static data, so AF can change the default yield in the future
- [ ] The default yield for the Raspberry pathway (5 tons/ac) is also added to static data, so AF can change it in the future

**Comment by fergmac** [comment]
Confirming that we are updating both the Blueberry and Raspberry default yield to 5 from 7?

![Image](https://github.com/bcgov/agri-nmp/assets/20193291/05d9c22a-1e54-46dd-8134-21379d0058a3)
![Image](https://github.com/bcgov/agri-nmp/assets/20193291/83324229-5941-44c8-b380-4ccc528c3658)

**Comment by joandrews** [comment]
@fergmac Yes, both should be 5 tons/ac, and both should be added to static data

**Comment by joandrews** [comment]
@fergmac I feel stupid for asking this, but which static data table has the berry yields? I assumed it would be in CropYields...

**Comment by acatchpole** [comment]
It is CropYields. The Amount = default crop yield. The CropId for blueberries and raspberries is 75 and 76 respectively. These entires do not yet exist in the prod database.

**Comment by joandrews** [comment]
Thank you!
